packages/ethereum/deployments/localhost/HoprChannels.json

```diff
@@ -1,6 +1,6 @@
 {
   "address": "0xDc64a140Aa3E981100a9becA4E685f962f0cF6C9",
-  "transactionHash": "0x1c2aea4e894923fa91440c1e1d85a9bb9833e33ede0758e41014ce14fa2e6abc",
```

Comment:
suggestion: ignore this file locally

Reply:
I think we need this file for our E2E tests, but am not 100% sure, I remember we included the file to fix something - but let's open an issue for it and investigate, unlike the production networks, I agree it shouldn't have to be committed

Reply:
Still commiting this file

Reply:
file is updated during deployment -> https://github.com/hoprnet/hoprnet/blob/f76cd09ecc15f49bfe764baf3a787044c46f4633/.github/workflows/deploy.yaml#L202 - I would wager that it's needed to be committed, reverted the change on my PR though